discord/ext/bridge/context.py

```diff
@@ -42,7 +42,7 @@ class BridgeContext(ABC):
     this class are meant to give parity between the two contexts, while still allowing for all of their functionality.
 
     When this is passed to a command, it will either be passed as :class:`BridgeExtContext`, or
-    :class:`BridgeApplicationContext`. Since they are two separate classes, it is quite simple to use :func:`isinstance`
+    :class:`BridgeApplicationContext`. Since they are two separate classes, it's easy to use `cts.is_app` 
```

Comment:
```suggestion
    :class:`BridgeApplicationContext`. Since they are two separate classes, it's easy to use the ``is_app`` attribute
```

Reply:
```suggestion
    :class:`BridgeApplicationContext`. Since they are two separate classes, it's easy to use the :attr:`BridgeContext.is_app` attribute.
```